apps/right-to-rent-check/behaviours/tenants.js

```diff
@@ -144,6 +145,10 @@ module.exports = fields => {
     locals(req, res) {
       const locals = super.locals(req, res);
       locals.tenants = req.sessionModel.get('tenants');
+      _.map(locals.tenants, (tenant) => {
+        tenant['tenant-dob'] = moment(tenant['tenant-dob'], ['YYYY-MM-DD', 'DD-MM-YYYY']).format('DD-MM-YYYY');
```

Comment:
this changes how the `tenant-dob` values are saved in `tenants`.
`tenants` is an object, so when you do `locals.tenants = req.sessionModel.get('tenants')` you are assigning a reference to the value in the model to `locals.tenants`. Which means any changes you make to `locals.tenants` are made to the `tenants` in the model.